### ISSUE TITLE
Maintenance: Archive orphan sample projects with Lambda Powertools for Java and move them to a proper location

### ISSUE THREAD

**Issue by machafer** [aws-powertools/powertools-lambda-java#1074]
### Summary

These two projects don’t have any maintainers and may cause confusion to developers looking for samples with Lambda Powertools for Java. They need to be archived and moved to a proper location.

- https://github.com/aws-samples/cdk-lambda-packaging-java
- https://github.com/aws-samples/serverless-webapp-mono-repo-ci-cd-java

### Why is this needed?

Better project governance and completeness. 

### Which area does this relate to?

Governance, Other

### Solution

Move to Powertools for Java project umbrella.

### Acknowledgment

- [X] This request meets [Lambda Powertools Tenets](https://awslabs.github.io/aws-lambda-powertools-java/#tenets)
- [ ] Should this be considered in other Lambda Powertools languages? i.e. [Python](https://github.com/awslabs/aws-lambda-powertools-python/), [TypeScript](https://github.com/awslabs/aws-lambda-powertools-typescript/)

**Comment by jeromevdl** [comment]
Definitely, it would be great to have a CDK sample (and maybe other frameworks: SST, serverless fwk).

Regarding the 2 projects you mention, I agree https://github.com/aws-samples/cdk-lambda-packaging-java does not provide much value and could be retrieved here (as the CDK sample). Regarding the other, looks too big and makes sense per se. It is not specifically related to powertools, it's using it but we cannot eat all the samples using powertools...

**Comment by msailes** [comment]
I think both of these samples were used in blog posts. I think they make sense in the point of time they were written. 

It would be ideal if they were shown to be points in time and that up to date powertools documentation / examples could be seen elsewhere.

**Comment by dreamorosi** [comment]
Both projects appear to have been removed and now return 404.
